tests/providers/apache/hive/hooks/test_hive.py

```diff
@@ -17,22 +17,13 @@
 # under the License.
 from __future__ import annotations
 
-import pytest
-
-from airflow import PY311
-
-if PY311:
-    pytest.skip(
-        "The tests are skipped because Apache Hive provider is not supported on Python 3.11",
-        allow_module_level=True,
-    )
-
```

Comment:
Seems like it should be resolved in https://github.com/apache/airflow/pull/32607
But lets have a look on tests

Reply:
When I check is pyhive supported Python 3.12, I've found that project doesn't supported anymore:
- https://github.com/dropbox/PyHive
- https://pypi.org/project/PyHive/

But this titles added two years ago by https://github.com/dropbox/PyHive/pull/423

Reply:
Yes. It stopped being supported, but it **should** work since we use pure_python_sasl implementation. Some day we might want to remove it I guess. Not sure when :D. Maybe when hive is moved to attic.